thunder/tests/test_interpreter.py

```diff
@@ -3305,3 +3305,23 @@ def fn(x):
     jfn = thunder.jit(fn)
     with pytest.raises(TypeError, match="got unexpected keyword arguments: incorrect_arg"):
         jfn(3, incorrect_arg=3)
+
+
+def test_class_setattr():
+    import torch
+    import thunder
+    import contextlib
```

Comment:
minor nit - move the imports to file level

Reply:
Thank you!